src/Connection.php

```diff
@@ -113,6 +113,11 @@ class Connection
      */
     protected static $multiByteSupport = null;
 
+    public static $validResources = array(
+        'socket' => true,
+        'stream' => true
+    );
+
```

Comment:
In my opinion the minimum supported version should be 5.5 because Ubuntu 14.04 LTS is still relevant